### PR TITLE
fix: upgraded versions.tf to include minor bumps from tpg v5

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

**Comment by g-awmalik** [comment]
@apeabody - PTAL?

File: modules/instance/versions.tf [hunk 1]
```diff
@@ -20,15 +20,15 @@ terraform {
...
     }
     null = {
       source  = "hashicorp/null"
       version = ">= 2.0"
     }
     template = {
-      source  = "hashicorp/template"
-      version = "~> 2.0"
+      source  = "hashicorp/template" # TODO: this provider is deprecated, replace with terraform 0.13+ templatefile function
+      version = "2.2.0"
```
Comment (apeabody) [inline modules/instance/versions.tf:31]: While `2.2.0` is pretty old, increasing the minimum provider version could be construed as a breaking change?

File: modules/template_files/versions.tf [hunk 2]
```diff
@@ -20,15 +20,15 @@ terraform {
...
     }
     local = {
       source  = "hashicorp/local"
       version = "~> 2.0"
     }
     template = {
-      source  = "hashicorp/template"
-      version = "~> 2.0"
+      source  = "hashicorp/template" # TODO: this provider is deprecated, replace with terraform 0.13+ templatefile function
+      version = "2.2.0"
```
Comment (apeabody) [inline modules/template_files/versions.tf:31]: Same as above

**Review by apeabody — APPROVED** [review]
@g-awmalik - LGTM, however a note on the `hashicorp/template` provider version.

File: modules/instance/versions.tf (hunk 1 above)
Comment (g-awmalik) [reply modules/instance/versions.tf:31]: It was already downloading 2.2 since it's been out for 3 years. I don't think it'll break anyone.

**Pull request merged**